### ISSUE TITLE
Design new loss function

### ISSUE THREAD

**Issue by leiemily** [lululxvi/deepxde#550]
Dear Prof. @lululxvi ,
I was trying to train a neural network based on the chain rule:
![image](https://user-images.githubusercontent.com/93039907/156483403-684dde5f-7035-42ed-98d0-c2574deced17.png)

The corresponding loss function may not either be the pde loss or data loss? How can I design this kind of loss function? Would you mind give me some advice?
Great thanks!

**Comment by leiemily** [comment]
Dear Prof. @lululxvi ,
Thanks for your reply!
![image](https://user-images.githubusercontent.com/93039907/157248383-de38604c-41a0-4793-86f4-9c9da3a576dc.png)

**Comment by lululxvi** [comment]
Not sure how to do it yet, because t is not an input, how can we compute dF/dt?

**Comment by leiemily** [comment]
Thanks for your reply! Maybe my idea just won't work. I will close the issue.
